lib/internal/bootstrap_node.js

```diff
@@ -72,22 +72,14 @@
         NativeModule.require('_third_party_main');
       });
 
-    } else if (process.argv[1] === 'debug') {
-      // Start the debugger agent
-      NativeModule.require('_debugger').start();
-
-    } else if (process.argv[1] === 'inspect') {
+    } else if (process.argv[1] === 'inspect' || process.argv[1] === 'debug') {
```

Comment:
Per my comment below, I think we should not yet alias `--debug` to `--inspect`. The reason for not yet aliasing `--debug` doesn't apply to `node debug`, but for consistency's sake, we should probably not yet alias `node debug` to `node inspect` either.